job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/properties/PropertiesContext.groovy

```diff
@@ -102,4 +102,16 @@ class PropertiesContext extends AbstractExtensibleContext {
             delegate.projectUrl(projectUrl ?: '')
         }
     }
+
+    /**
+     * Configures the Build Failure Analyzer.
+     *
+     * @since 1.41
+     */
+    @RequiresPlugin(id = 'build-failure-analyzer', minimumVersion = '1.13')
+    void scannerJobProperty(Boolean doNotScan = false) {
```

Comment:
Methods in properties context should not be named xxxProperty. It's obvious that they are properties from the context. Rename this to `buildFailureAnalyzer`.

Use `boolean` instead of `Boolean`. `Boolean` values can be `null` which does not make any sense in this case. 

Set the default value to true, otherwise `scannerJobProperty()` would disable the scanner which is counterintuitive.